packages/primitives/button/test/index.ts

```diff
@@ -0,0 +1,56 @@
+import test from 'tape'
+import { getAppPage } from 'foreal'
+
+type TFixtures = 'bare-bones' | 'disabled'
+const getFixture = (filename: TFixtures) => `./fixtures/${filename}.tsx`
+
+const PACKAGE = '@primitives/button'
+
+test(`${PACKAGE} has proper role`, async (t) => {
+  const entryPointPath = require.resolve(getFixture('bare-bones'))
+
+  const page = await getAppPage({
+    entryPointPath,
+  })
+
+  const el = await page.$('#btn')
+
+  const a11ySnapBtn = await page.accessibility.snapshot({ interestingOnly: false, root: el })
```

Comment:
> page.accessibility

🤯

Reply:
it works for me without setting `interestingOnly`, do we need it?

Reply:
not here, `interestingOnly` defaults to `false`